deploy/service-bus/create.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+require('dotenv').config()
+const azure = require('azure')
+const sbService = azure.createServiceBusService()
+const queues = require('./queues-topics.json').queues
+const fiveGigabytes = 5120
+const fourteenDays = 'P14D'
+const fiveMinutes = 'PT5M'
+const oneDay = 'P1D'
+
+const defaultQueueOptions = {
+  MaxSizeInMegabytes: fiveGigabytes,
+  DefaultMessageTimeToLive: fourteenDays,
+  LockDuration: fiveMinutes,
+  RequiresDuplicateDetection: true,
+  DeadLetteringOnMessageExpiration: true,
+  DuplicateDetectionHistoryTimeWindow: oneDay,
+  EnablePartitioning: false,
+  RequiresSession: false
+}
+
+const createQueue = (queueName, queueOptions) => (new Promise((resolve, reject) => {
+  sbService.createQueueIfNotExists(queueName, queueOptions, function (error) {
+    if (!error) {
+      console.log(`${queueName} queue created`)
+      resolve()
+    } else {
+      reject(error)
+    }
+  })
+}))
+
+async function main () {
+  try {
+    const promises = queues.map(q => createQueue(q, defaultQueueOptions))
+    await Promise.all(promises)
+  } catch (error) {
+    throw new Error(`Error creating queue: ${error.message}`)
```

Comment:
Not sure that catching the error only to throw a new error is adding much.   Do we really need the try/catch here?

Reply:
good catch (see what i did there)